internal/fabric/bccsp/utils/keys.go

```diff
@@ -49,22 +49,22 @@ func PEMtoPrivateKey(raw []byte, pwd []byte) (interface{}, error) {
 
 	// TODO: derive from header the type of the key
 
-	if x509.IsEncryptedPEMBlock(block) {
-		if len(pwd) == 0 {
-			return nil, errors.New("Encrypted Key. Need a password")
-		}
-
-		decrypted, err := x509.DecryptPEMBlock(block, pwd)
-		if err != nil {
-			return nil, fmt.Errorf("Failed PEM decryption [%s]", err)
-		}
-
-		key, err := DERToPrivateKey(decrypted)
-		if err != nil {
-			return nil, err
-		}
-		return key, err
-	}
+	//if x509.IsEncryptedPEMBlock(block) {
```

Comment:
if I remember correctly the tj gm lib also used pcks8 as pem export standard. so this pass word check still needed. and to pass CI check, we need a switch here for tape use  switch between gm and ecdsa.